scripts/core/editor3/components/Editor3.jsx

```diff
@@ -0,0 +1,226 @@
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Editor, EditorState, RichUtils, ContentState} from 'draft-js';
+
+export class Editor3 extends React.Component {
+    /**
+     * @ngdoc react component
```

Comment:
@gbbr Do you have any example how to document React component
Seams that right now .jsx files are ignored at all when the docs are generated.
If the filter for dgeni-alive grunt task is changed in order to include jsx files I get the error:
error:   Error: No file reader found for /home/ioan/sd/superdesk-client-core/scripts/core/editor3/components/Editor3.jsx